src/App.js

```diff
@@ -1,28 +1,32 @@
 import React from 'react'
 import { Route, Switch } from 'react-router-dom'
-import Motives from './Motives/Motives'
-import QuestionnairePage from './Questionnaire/QuestionnairePage'
+import MotivesEntry from './Motives/MotivesEntry'
```

Comment:
if MotivesEntry is a page just like your QuestionnaireIntroPage you could also name it "MotivesEntryPage" to make the naming more consistent - same goes for MotivesIntro and PersonalityStyleInfo

Reply:
That's a good point. Currently I only added the "Page" components to parse the URL parameters. MotivesEntry doesn't need any params. 
I added a note in my project board to review the structure in the next sprint.